Update BackendInfo documentation (#102)
CQCL/pytket: manual/manual_noise.rst

Some devices expose error characterisation information through
their programming interface. When available, :py:class:`Backend`
objects will populate a :py:class:`BackendInfo` object with this information. 

A :py:class:`BackendInfo` object contains a variety of characterisation information supplied by hardware providers.   

> Some information, including gate error rates, are stored in prenamed attributes

Reviewer: ```suggestion
Some information, including gate error rates, is stored in attributes with specific names.
```

.. jupyter-input::

    from pytket.extensions.qiskit import IBMQBackend

    backend = IBMQBackend("ibmq_santiago")
    print(backend.backend_info.averaged_node_gate_errors)

.. jupyter-output::

   {node[0]: 0.0002186159622502225, 
    node[1]: 0.0002839221599849252, 
    node[2]: 0.00014610243862697218, 
    node[3]: 0.00015814094160059136, 
    node[4]: 0.00013411930305754117}    

> While other information is stored in a generic dictionary and is accessible through the :py:class:`Backend` `characterisation` member.

Reviewer: ```suggestion
Other information is stored in a generic dictionary and is accessible through the :py:class:`Backend` `characterisation` member.
```
